sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -7814,36 +7817,43 @@ public static partial class PageBlob
                 {
                     case 201:
                     {
-                        // Create the result
-                        Azure.Storage.Blobs.Models.PageInfo _value = new Azure.Storage.Blobs.Models.PageInfo();
 
                         // Get response headers
                         string _header;
+                        Azure.Core.Http.ETag eTag = default;
```

Comment:
Other variables are defined with `_` prefix. While I like how they look without prefix we should be consistent.

Reply:
I'm not actually sure if we want that here.  I prefixed the "well known variables" with an `_` to avoid conflicts, but these are the sort of values that could cause conflicts with the `_` variables.  For example, if somebody adds a struct with a property called `xml`, we wouldn't want that to conflict with our "well known variable" `_xml`.  I think this should probably change back what you originally had.